custom-words.txt

```diff
@@ -519,6 +519,7 @@ connectionstrings
 conntrack
 consistencygroup
 consumergroups
+containerd
```

Comment:
word is used in documentation
containerd is a common container runtime
-> http://containerd.io